README.md

```diff
@@ -68,37 +78,9 @@ See [Configuring Crawlers](#configuring-crawlers) to begin crawling web content.
 
 See [CONFIG.md](docs/CONFIG.md) for in-depth details on Crawler configuration files.
 
-Once you have a Crawler configured, you can validate the domain(s) using the CLI.
+### CLI Commands
```

Comment:
We should make it abundantly clear early on (perhaps in the How it works section) that the user controls the crawler using a command line interface.